CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2022-10-24
+
+Monitor the release status by regions at [AKS-Release-Tracker](https://releases.aks.azure.com/).
+
+### Announcements
+
+* AKS is retiring `v1.22.x` on December 4th 2022. Please upgrade your clusters to `v1.23` and above.
+* Starting with Kubernetes 1.25, the following changes will be made default:
+  * Ubuntu 22.04 for x86, AMD and ARM64 architectures will be the default host.
+* Some AKS labels are being deprecated with the Kubernetes 1.26 release. Update your AKS labels to the recommended substitutions. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation. `beta.kubernetes.io/arch=` and `beta.kubernetes.io/os=`  are still applied by kubelet in kubernetes code.
+* Docker is no longer supported as a container runtime on Windows. Follow these [steps](https://docs.microsoft.com/azure/aks/learn/quick-windows-container-deploy-cli#:~:text=Upgrade%20an%20existing%20Windows%20Server%20node%20pool%20to%20containerd) in our documentation to upgrade your Kubernetes cluster to change your container runtime to containerd.
+* Kubernetes 1.25 is in preview. We support Kubernetes 1.25.2.
+
+### Release notes
+
+* Features
+  * Enables v5 SKUs with Azure CNI Overlay (public preview) - https://aka.ms/aks/overlay
+
+* Behavior Changes
+  * Stop clusters with 0 nodes for more than 30 days as documented in [support policy](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters)
```

Comment:
This is a pretty large change for customers since we're now enforcing this policy. We may want to move it to announcements. I'm also working on sending out comms for this as well

Reply:
Can we change the announcement to: 
AKS will be enforcing the [de-allocated clusters](https://learn.microsoft.com/en-us/azure/aks/support-policies#stopped-or-de-allocated-clusters) policy which specifies that manually de-allocating clusters renders the cluster out of support. Starting November 1, 2022 clusters with zero nodes will be stopped after 30 days.

Reply:
Internal subs will be shutdown after 1d - thoughts on if that should be included in release notes?

Reply:
I dont see it as a problem given we dont have another place to put 'internal only' announcements.